packages/xod-arduino/src/utils.js

```diff
@@ -0,0 +1,31 @@
+import * as R from 'ramda';
+
+import * as XP from 'xod-project';
+import { def } from './types';
+
+// :: x -> Number
+export const toInt = R.flip(parseInt)(10);
+
+export const kebabToSnake = R.replace(/-/g, '_');
+
+// foo(number,string) -> foo__number__string
+export const sanitizeTypeSpecification = R.compose(
+  R.replace(/\(|,/g, '__'),
+  R.replace(')', '')
+);
+
+export const createPatchNames = def(
+  'createPatchNames :: PatchPath -> { owner :: String, libName :: String, patchName :: String }',
```

Comment:
Not quite clear what does it do at a first sight. A comment with example and more precise name can help.

I’m not sure whether the function is useful on its own. It returns a quite strange intermediate structure. Will it be better if it will convert to a namespace name as the last step so that the intermediate type goes away?